lms/views/api/exceptions.py

```diff
@@ -178,23 +180,23 @@ def api_error(self):
 
         1. The `error_code` attribute will be used as the "error_code" field in
            the response's JSON body
-
         2. The response's HTTP status will be 400
-
         3. If the exception also has a `details` attribute this will be used as
            "details" field in the response's body
 
-        If the exception does not have an `error_code` attribute then:
+        You can use `SerializableError` or a child if you'd like to trigger
+        this conveniently.
 
+        If the exception does not have an `error_code` attribute then:
         1. The response's HTTP status will be 500
-
         2. A fixed string (see below) will be used as the "message" field in
            the response's JSON body
         """
 
         if hasattr(self.context, "error_code"):
             return ErrorBody(
                 error_code=self.context.error_code,
+                message=getattr(self.context, "message", None),
```

Comment:
Who's consuming this message? Is it mostly for being more human-readable on the let's say chrome devtools?

Reply:
This is what you see when you are in the file picker.